gatsby-config.js

```diff
@@ -468,6 +468,13 @@ module.exports = {
         name: "integrations",
       },
     },
+    {
+      resolve: "gatsby-source-filesystem",
+      options: {
+        name: "images",
```

Comment:
Update the name to "integration-images" as we already have a source collection with the name of images.